ui/src/lib/yjs/useYjsStore.ts

```diff
@@ -28,67 +28,77 @@ export default ({
 }) => {
   const { toast } = useToast();
   const t = useT();
-
   const [currentProject] = useCurrentProject();
   const { createDeployment, useUpdateDeployment } = useDeployment();
-
-  const yWebSocketRef = useRef<WebsocketProvider | null>(null);
-  useEffect(() => () => yWebSocketRef.current?.destroy(), []);
-
+  const managerRef = useRef<SocketYjsManager | null>(null);
   const [undoManager, setUndoManager] = useState<Y.UndoManager | null>(null);
 
-  const [{ yWorkflows, currentUserClientId, undoTrackerActionWrapper }] =
-    useState(() => {
-      const yDoc = new Y.Doc();
-      const { websocket, websocketToken } = config();
-      if (workflowId && websocket && websocketToken) {
-        yWebSocketRef.current = new WebsocketProvider(
-          websocket,
-          workflowId,
-          yDoc,
-          { params: { token: websocketToken } },
-        );
-      }
+  // Initialize store and connect
+  const [{ yWorkflows, currentUserClientId }] = useState(() => {
+    const doc = new Y.Doc();
+    const yWorkflows = doc.getArray<YWorkflow>("workflows");
 
-      const yWorkflows = yDoc.getArray<YWorkflow>("workflows");
+    // Initialize with default workflow if empty
+    if (yWorkflows.length === 0) {
```

Comment:
This isn't necessary since the doc will always be new, and the useState's initial values (aka this code) will never re-run.